ml-prims/test/reduce.h

```diff
@@ -20,65 +20,73 @@
 #include "linalg/unary_op.h"
 #include "linalg/cublas_wrappers.h"
 
-
 namespace MLCommon {
 namespace LinAlg {
 
-template <typename Type>
+template<typename Type>
 __global__ void naiveCoalescedReductionKernel(Type *dots, const Type *data,
-                                              int D, int N) {
-  Type acc = (Type)0;
-  int rowStart = threadIdx.x + blockIdx.x * blockDim.x;
-  if (rowStart < N) {
-    for (int i = 0; i < D; ++i) {
-        acc += data[rowStart * D + i] * data[rowStart * D + i];
-    }
-    dots[rowStart] = 2*acc;
-  }
+		int D, int N) {
+	Type acc = (Type) 0;
+	int rowStart = threadIdx.x + blockIdx.x * blockDim.x;
+	if (rowStart < N) {
+		for (int i = 0; i < D; ++i) {
```

Comment:
@oyilmaz-nvidia  Your text editor replaced all nice 2 spaced indentation with tabs.

Reply:
fixed this in my commit